cmd/admin/controller/account/guest.go

```diff
@@ -0,0 +1,118 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+package account
+
+import (
+	"log"
+	"net/http"
+	"time"
+)
+
+import (
+	"github.com/dgrijalva/jwt-go"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/pkg/errors"
+)
+
+import (
+	"github.com/dubbogo/pixiu-admin/cmd/admin/controller"
+	"github.com/dubbogo/pixiu-admin/cmd/admin/controller/auth"
+	"github.com/dubbogo/pixiu-admin/pkg/logic/account"
+	"github.com/dubbogo/pixiu-admin/pkg/utils"
+)
+
+func Register(c *gin.Context) {
+	username := c.PostForm("username")
+	password := c.PostForm("password")
+
+	// username password Format check
```

Comment:
Is this commented out code necessary?

Reply:
The web-side project was implemented by another developer, so the implementation of this function has not been discussed yet